.github/workflows/dotnetcore.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
```

Comment:
You can remove `dotnet-version` (and the entire `with` section). `actions/setup-dotnet@v1` reads the version from `global.json`

Reply:
Adressed in 6c9d69126b7cd3825d4b1afe219814e5f95b1f51